config/initializers/content_security_policy.rb

```diff
@@ -36,7 +36,7 @@ def host_to_url(str)
     p.worker_src  :self, :blob, assets_host
   else
     p.connect_src :self, :data, :blob, assets_host, media_host, Rails.configuration.x.streaming_api_base_url
-    p.script_src  :self, assets_host, :unsafe_eval
+    p.script_src  :self, assets_host, "'unsafe-wasm-eval'"
```

Comment:
Wait, according to the issue discussion, it's `wasm-unsafe-eval`, not `unsafe-wasm-eval`. Which is it?